.pipelines/templates/PackageBuild.yml

```diff
@@ -18,6 +18,18 @@ parameters:
     type: string
     default: ""
 
+  - name: dailyBuildID
+    type: string
+    default: ""
+    displayName: "Daily build ID"
+
+  - name: isDailyBuildUpdateManifests
```

Comment:
Please keep parameters in alphabetical order as well. Thanks!

Reply:
Updated